plan/config_test.go

```diff
@@ -116,3 +126,46 @@ func TestParseBehavior(t *testing.T) {
 		assert.Equal(t, tt.want, parseBehavior(tt.give))
 	}
 }
+
+func TestParseSkipBehavior(t *testing.T) {
+	tests := []struct {
+		give       string
+		wantFilter []Filter
+		wantKey    string
+	}{
+		{
+			give: "foo=client=c+server=b",
```

Comment:
perhaps give descriptions for each of these table test cases (and use `t.Run(tt.desc, func(t) {})`), e.g., "skip all cases in the 'foo' behavior where the CLIENT is 'foo' and SERVER is 'b') (did I get that right?)